src/libstd/sys/windows/fs.rs

```diff
@@ -12,6 +12,8 @@ use crate::sys::handle::Handle;
 use crate::sys::time::SystemTime;
 use crate::sys::{c, cvt};
 use crate::sys_common::FromInner;
+#[cfg(target_os = "uwp")]
+use libc::c_void;
```

Comment:
It might be easier to just refer to `libc::c_void` directly below to avoid the `#[cfg]`'d import here

Reply:
Very fair point, will do